lib/rails_event_store/event.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_support/inflector'
```

Comment:
yeah, missed that. but anyway I'm moving this to ruby_event_store

expect new release soon